apps/accounts/tests/test_views.py

```diff
@@ -532,34 +537,58 @@ def test_identify_success_no_account_with_path(self):
             'state': 'the-right-blob:{next_path}'.format(
                 next_path=base64.urlsafe_b64encode('/go/here')),
         })
-        assert response.status_code == 302
-        assert response['location'] == absolutify(reverse('users.edit'))
+        # This 302s because the user isn't logged in due to mocking.
+        self.assertRedirects(
+            response, reverse('users.edit'), target_status_code=302)
         self.fxa_identify.assert_called_with('codes!!', config=FXA_CONFIG)
         assert not self.login_user.called
         self.register_user.assert_called_with(mock.ANY, identity)
 
-    def test_identify_success_exists_logs_user_in(self):
+    def test_identify_success_exists_logs_user_in_migrated(self):
+        user = UserProfile.objects.create(email='real@yeahoo.com', fxa_id='10')
+        identity = {'email': 'real@yeahoo.com', 'uid': '9001'}
+        self.fxa_identify.return_value = identity
+        response = self.client.get(
+            self.url, {'code': 'code', 'state': 'the-right-blob'})
+        self.assertRedirects(response, '/en-US/firefox/')
+        self.login_user.assert_called_with(mock.ANY, user, identity)
+        assert not self.register_user.called
+
+    def test_identify_success_exists_logs_user_in_with_path_migrated(self):
+        user = UserProfile.objects.create(email='real@yeahoo.com', fxa_id='10')
+        identity = {'email': 'real@yeahoo.com', 'uid': '9001'}
+        self.fxa_identify.return_value = identity
+        response = self.client.get(self.url, {
+            'code': 'code',
+            'state': 'the-right-blob:{next_path}'.format(
```

Comment:
I find `the-right-blob` distracting and a little confusing. Is there a way to make it somewhat realistic?